src/Umbraco.Infrastructure/Packaging/PackageDataInstallation.cs

```diff
@@ -408,25 +411,27 @@ public IReadOnlyList<IMediaType> ImportMediaTypes(IEnumerable<XElement> docTypeE
         #region DocumentTypes
 
         public IReadOnlyList<IContentType> ImportDocumentType(XElement docTypeElement, int userId)
-            => ImportDocumentTypes(new[] { docTypeElement }, userId);
+            => ImportDocumentTypes(new[] { docTypeElement }, userId, new List<EntityContainer>());
 
         /// <summary>
         /// Imports and saves package xml as <see cref="IContentType"/>
         /// </summary>
         /// <param name="docTypeElements">Xml to import</param>
         /// <param name="userId">Optional id of the User performing the operation. Default is zero (admin).</param>
+        /// <param name="entityContainersInstalled">List of entity containers installed by the package to be populated with those created in installing data types.</param>
         /// <returns>An enumerable list of generated ContentTypes</returns>
-        public IReadOnlyList<IContentType> ImportDocumentTypes(IEnumerable<XElement> docTypeElements, int userId)
-            => ImportDocumentTypes(docTypeElements.ToList(), true, userId, _contentTypeService);
+        public IReadOnlyList<IContentType> ImportDocumentTypes(IEnumerable<XElement> docTypeElements, int userId, List<EntityContainer> entityContainersInstalled)
+            => ImportDocumentTypes(docTypeElements.ToList(), true, userId, entityContainersInstalled, _contentTypeService);
 
         /// <summary>
         /// Imports and saves package xml as <see cref="IContentType"/>
         /// </summary>
         /// <param name="unsortedDocumentTypes">Xml to import</param>
         /// <param name="importStructure">Boolean indicating whether or not to import the </param>
         /// <param name="userId">Optional id of the User performing the operation. Default is zero (admin).</param>
+        /// <param name="entityContainersInstalled">List of entity containers installed by the package to be populated with those created in installing data types.</param>
         /// <returns>An enumerable list of generated ContentTypes</returns>
-        public IReadOnlyList<T> ImportDocumentTypes<T>(IReadOnlyCollection<XElement> unsortedDocumentTypes, bool importStructure, int userId, IContentTypeBaseService<T> service)
+        public IReadOnlyList<T> ImportDocumentTypes<T>(IReadOnlyCollection<XElement> unsortedDocumentTypes, bool importStructure, int userId, List<EntityContainer> entityContainersInstalled, IContentTypeBaseService<T> service)
```

Comment:
This is a breaking change, we need to old signature reintroduced and obsoleted

Reply:
Have restored the old signature.